core/shared/src/main/scala/tech/ant8e/uuid4cats/TypeID.scala

```diff
@@ -202,6 +202,11 @@ object TypeID {
       case _             => false
     }
 
+    override def hashCode(): Int = {
+      val prime = 31
+      prime * (prime + prefix.hashCode) + uuid.hashCode
```

Comment:
It would be nice to do the calculation only once in a lazy val and just return that in the function

Reply:
I've introduced a lazy val.

Reply:
👍